dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -968,6 +968,11 @@
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_PROCESSING_SKIPPOSITIONS
       # skipPositions: ""
 
+      # Configures the rate at which a partition leader checks for expired scheduled tasks such as the due date checker.
+      # The default value is 1 second. Use a lower interval to potentially decrease delays between requested and actual execution of scheduled tasks.
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_PROCESSING_SCHEDULEDTASKCHECKINTERVAL
```

Comment:
:question: Should we put a warning about a possible performance issue if you put it too low? Or in the docs? :thinking: Essentially, I'm trying to anticipate a user saying "oh well, I don't want delays right?" :smile:

Reply:
Alternative is putting an enforced lower bound, e.g. 100ms, 200ms, something.

Reply:
I've added a small note for this :+1: